openapi/openapi.yaml

```diff
@@ -934,6 +934,30 @@ paths:
             default: false
           description: >
             Do not use cached results, always run adviser.
+        - name: github_event_id
```

Comment:
Not sure if this is going to be required.

Reply:
maybe we can leave it for future extension of other events?

Reply:
Let's call it `github_event_type` as we are extending GitHub Event pool: https://developer.github.com/v3/activity/events/types/